src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -4356,17 +4357,19 @@ void ExportMusicXml::chord(Chord* chord, staff_idx_t staff, const std::vector<Ly
             notations.tag(m_xml, tieBack);
             m_xml.tag("tied", { { "type", "stop" } });
         }
+
+        const LaissezVib* laissezVib = note->laissezVib();
+        if (laissezVib && ExportMusicXml::canWrite(laissezVib)) {
+            notations.tag(m_xml, laissezVib);
+            m_xml.tag("tied", { { "type", "let-ring" } });
+        }
+
         const Tie* tieFor = note->tieFor();
-        if (tieFor && ExportMusicXml::canWrite(tieFor)) {
+        if (tieFor && !laissezVib && ExportMusicXml::canWrite(tieFor)) {
```

Comment:
This way all styling of lv ties gets lost. Better to adjust the previous clause to just exchange `type="start"` with `type="let-ring"`.